packages/application-generic/src/usecases/get-decrypted-integrations/get-decrypted-integrations.usecase.ts

```diff
@@ -21,6 +21,43 @@ export class GetDecryptedIntegrations {
     private getFeatureFlag: GetFeatureFlag
   ) {}
 
+  private async getNovuIntegrations(command: GetDecryptedIntegrationsCommand) {
+    let novuIntegrations: IntegrationEntity[] = [];
+    const novuEmailIntegration = await this.getNovuIntegration.execute(
+      GetNovuIntegrationCommand.create({
+        channelType: ChannelTypeEnum.EMAIL,
+        organizationId: command.organizationId,
+        environmentId: command.environmentId,
+        userId: command.userId,
+        ignoreActiveCount: true,
+      })
+    );
+
+    if (novuEmailIntegration) {
+      novuIntegrations.push(novuEmailIntegration);
+    }
+
+    const novuSmsIntegration = await this.getNovuIntegration.execute(
+      GetNovuIntegrationCommand.create({
+        channelType: ChannelTypeEnum.SMS,
+        organizationId: command.organizationId,
+        environmentId: command.environmentId,
+        userId: command.userId,
+        ignoreActiveCount: true,
+      })
+    );
+
+    if (novuSmsIntegration) {
+      novuIntegrations.push(novuSmsIntegration);
+    }
+
+    if (command.active) {
+      novuIntegrations = novuIntegrations.filter((el) => el.active);
+    }
```

Comment:
fix for another bug: `use-case returns integrations that are not active --> for ex. novu-email integration`